src/fromager/bootstrapper.py

```diff
@@ -351,17 +351,18 @@ def _build_sdist(
                 )
         except Exception as err:
             logger.warning(f"failed to build source distribution: {err}")
-        return (sdist_filename, build_env)
+            raise
```

Comment:
Have you tried this code on our downstream builds? I wonder if this change in error handling is going to cause any issues?

Reply:
No, I have not tested this with our downstream builds. I can undo the change and create a ticket to investigate.